pkg/users/account-details.js

```diff
@@ -352,6 +352,7 @@ export const AccountGroupsSelect = ({ name, loggedIn, groups, setError }) => {
     const [isOpenGroup, setIsOpenGroup] = useState(false);
     const [selected, setSelected] = useState();
     const [primaryGroupName, setPrimaryGroupName] = useState();
+    const [loading, setLoading] = useState(true);
```

Comment:
Just in case you need to re-push: The prefix of this commit message should be "users:", not "test:"